src/csync/vio/csync_vio_local_win.cpp

```diff
@@ -158,8 +161,12 @@ std::unique_ptr<csync_file_stat_t> csync_vio_local_readdir(csync_vio_handle_t *d
           return nullptr;
       }
   }
+  auto path = c_utf8_from_locale(handle->ffd.cFileName);
```

Comment:
I just thought this was a leak, maybe I should have renamed it to `c_utf8_bytearray_from_locale` when I changed the return type.